scripts/compile.sh

```diff
@@ -67,12 +67,19 @@ function compile_test {
     $CXX -o ./build/${PREFIX}_${name} $OPT $ARCH ./build/${PREFIX}_main.o ./build/${PREFIX}_test_${name}.o $LDFLAGS
 }
 
+function compile_test_with_main {
+    local name=$1
+    $CXX -o ./build/${PREFIX}_test_${name}.o $OPT $DISASSEMBLY $ARCH $SYSROOT $CXXFLAGS -c test/test_${name}.cpp
+    $CXX -o ./build/${PREFIX}_${name} $OPT $ARCH ./build/${PREFIX}_test_${name}.o $LDFLAGS
+}
```

Comment:
Don't you need to define `JC_TEST_USE_DEFAULT_MAIN` here?

Reply:
Defined in test_minimal.cpp.